ome/figure
# Final bug fixes

This fixes a couple of minor bugs.

To test:
 - Log in as a user who has no figure files (owns none and can't access any others' files)
 - Should see "You have no figures" message below (console shouldn't show a 500 error)
 - Open "Draw ROIs" dialog for a panel. Select the Arrow tool and hold down shift key when starting to draw the Arrow. 
 - Arrow should be constrained to vertical or horizontal.

![screen shot 2015-11-21 at 22 44 59](https://cloud.githubusercontent.com/assets/900055/11320889/9c64b88c-90a1-11e5-8383-651b5af183a6.png)

Fixes: 
```
ApiUsageException: exception ::omero::ApiUsageException
{
   serverStackTrace = ome.conditions.ApiUsageException: unexpected end of subtree [select i.id from ome.model.core.Image i where i.id 
```
